verifiable/did_doc.go

```diff
@@ -0,0 +1,20 @@
+package verifiable
+
+// DIDDocument defines current supported did doc model.
+type DIDDocument struct {
+	Context []string  `json:"context"`
```

Comment:
In general we use `@context` for context field

Reply:
that's right.

Reply:
done